MiauAPI/Models/Requests/CreatedPetRequest.cs

```diff
@@ -0,0 +1,16 @@
+using MiauDatabase.Entities;
+using MiauDatabase.Enums;
+
+namespace MiauAPI.Models.Requests;
+
+/// <summary>
+/// Represents a request for creation of a new pet.
+/// </summary>
+/// <param name="User">The pet's user.</param>
+/// <param name="Name">The name of the pet.</param>
+/// <param name="Type">The type of the pet.</param>
+/// <param name="Gender">The gender of the pet.</param>
+/// <param name="Breed">The breed of the pet.</param>
+/// <param name="ImageFileUrl">This pet's image location in the file system.</param>
+/// <param name="DateOfBirth">This pet's date of birth.</param>
+public sealed record CreatedPetRequest(UserEntity User, string Name, PetType Type, PetGender Gender, string? Breed, string? ImageFileUrl, DateTime DateOfBirth);
```

Comment:
Imagens serão salvas localmente no servidor, portanto não tem como o cliente saber de antemão onde o arquivo será guardado. Fiz [uma pesquisa](https://www.appsloveworld.com/image-upload-in-net-core-web-api) e parece que `IFormFile` é o tipo correto pra fazer upload de arquivos, o que inclui imagens.

A imagem então será salva localmente e seu endereço no sistema de arquivos será salvo no banco de dados, para futura referência. Esse endereço nunca será exposto ao usuário.